modules/dag-l1/src/main/scala/io/constellationnetwork/dag/l1/Main.scala

```diff
@@ -158,6 +159,12 @@ object Main
         )
         .asResource
 
+      alignment = Alignment.make[IO, GlobalSnapshotStateProof, GlobalIncrementalSnapshot, GlobalSnapshotInfo, Run](
```

Comment:
just a small thing I noticed, but it is a little confusing to have a `class Alignment` and a `sealed trait Alignment` both defined in `dag-l1`  ([trait](https://github.com/Constellation-Labs/tessellation/blob/ce02a6fa52d0016849c13cc9e1666b682ae82eaa/modules/dag-l1/src/main/scala/io/constellationnetwork/dag/l1/domain/snapshot/programs/SnapshotProcessor.scala#L421))

Reply:
Good catch. I've just changed it to `GlobalSnapshotAlignment`